enclave/src/main/java/org/hyperledger/besu/enclave/RequestTransmitter.java

```diff
@@ -27,5 +27,9 @@
       String endpoint,
       ResponseBodyHandler<T> responseBodyHandler);
 
-  <T> T get(String endpoint, ResponseBodyHandler<T> responseBodyHandler);
+  <T> T get(
+      String mediaType,
+      String content,
```

Comment:
This is needed because we have to set the contentType and the content when we are doing the /receive with Tessera, but we can't set it for the /upcheck with Orion or Tessera.

Reply:
Is the content-type dynamic? Isn't this something that can be set when instantiating the implementation of RequestTransmitter?